pkgs/development/libraries/redis-plus-plus/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, stdenv, fetchFromGitHub, cmake, hiredis }:
+
+stdenv.mkDerivation rec {
+  pname = "redis-plus-plus";
+  version = "1.3.3";
+
+  src = fetchFromGitHub {
+    owner = "sewenew";
+    repo = "redis-plus-plus";
+    rev = "${version}";
+    sha256 = "sha256-k4q5YbbbKKHXcL0nndzJPshzXS20ARz4Tdy5cBg7kMc=";
+  };
+
+  nativeBuildInputs = [ cmake ];
+  propagatedBuildInputs = [ hiredis ];
```

Comment:
Why is this propagated?

Reply:
A project that wants to use the redis-plus-plus lib will need to have hiredis available to compile.

When testing this locally I could either propagate hiredis or explicitly declare hiredis as a dependency in the flake for my c++ project. I thought propagating was the cleaner solution